spec/system/task_spec.rb

```diff
@@ -1,24 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
`association`の定義をつけると`project_id`の記載は省略できますね。
```suggestion
  let(:task) { create(:task) }
```

Reply:
[7ce2913](https://github.com/MocoAristo/rspec_app_exam/pull/1/commits/7ce29136fbffa0d9cf3ea3a574d55cb51387e128)
でassociationを削除しました。